app/metrics.yaml

```diff
@@ -1667,6 +1667,19 @@ preferences:
     notification_emails:
       - android-probes@mozilla.com
     expires: "2022-11-01"
+  search_term_groups_enabled:
```

Comment:
Is this tracking if the feature is ON, or if the feature is SEEN by the user? (I can't remember what we defined for this)

Reply:
This is just to track if the feature is on.